packages/checkbox/src/checkbox-group.vue

```diff
@@ -9,7 +9,9 @@
     mixins: [Emitter],
 
     props: {
-      value: {}
+      value: {},
+      minimum: String,
+      maximum: String
```

Comment:
Ok, in fact I had used `Number`, but I was stacked when passing props.

I found [this](https://github.com/vuejs/vue/issues/1704), and now it's ok.

I rename also in `min` and `max`.